internal/test/helpers/envtest.go

```diff
@@ -60,7 +62,15 @@ import (
 )
 
 func init() {
+	// Set log level 5 as default for testing (default for prod is 2).
```

Comment:
Log level 5 gives us more verbose logs of our controllers, but also a lot of useful logs from CR around Reconciles